prime-router/src/main/kotlin/history/SubmissionHistory.kt

```diff
@@ -410,6 +414,23 @@ class DetailedSubmissionHistory(
         }
     }
 
+    /**
+     * Enrich a parent detailed history with details from the convert action.
+     * Add errors, and warnings, to the history details.
+     *
+     * @param descendant route action that will be used to enrich
+     */
+    private fun enrichWithConvertAction(descendant: DetailedSubmissionHistory) {
+        require(
```

Comment:
I was going to ask if we needed to unit test the negative case here, but I think based on my understanding this is technically not possible for those conditions to be falsified in the current setup, right?